contracts/minter-suite/Minters/MinterDAExpSettlementV2.sol

```diff
@@ -0,0 +1,1342 @@
+                    )
+                ),
+            "Only Artist or Admin ACL"
+        );
+        _;
+    }
+
+    // modifier to restrict access to only AdminACL allowed calls
+    // @dev defers which ACL contract is used to the core contract
+    modifier onlyCoreAdminACL(bytes4 _selector) {
+        require(
+            genArtCoreContract_Base.adminACLAllowed(
+                msg.sender,
+                address(this),
+                _selector
+            ),
+            "Only Core AdminACL allowed"
+        );
+        _;
+    }
+
+    modifier onlyArtist(uint256 _projectId) {
+        require(
+            (msg.sender ==
+                genArtCoreContract_Base.projectIdToArtistAddress(_projectId)),
+            "Only Artist"
+        );
+        _;
+    }
+
+    /**
+     * @notice Initializes contract to be a Filtered Minter for
+     * `_minterFilter`, integrated with Art Blocks core contract
+     * at address `_genArt721Address`.
+     * @param _genArt721Address Art Blocks core contract address for
+     * which this contract will be a minter.
+     * @param _minterFilter Minter filter for which
+     * this will a filtered minter.
+     */
+    constructor(
+        address _genArt721Address,
+        address _minterFilter
+    ) ReentrancyGuard() MinterBase(_genArt721Address) {
+        genArt721CoreAddress = _genArt721Address;
+        // always populate immutable engine contracts, but only use appropriate
+        // interface based on isEngine in the rest of the contract
+        genArtCoreContract_Base = IGenArt721CoreContractV3_Base(
+            _genArt721Address
+        );
+        minterFilterAddress = _minterFilter;
+        minterFilter = IMinterFilterV0(_minterFilter);
+        require(
+            minterFilter.genArt721CoreAddress() == _genArt721Address,
+            "Illegal contract pairing"
+        );
+    }
+
+    /**
+     * @notice Syncs local maximum invocations of project `_projectId` based on
+     * the value currently defined in the core contract. Also syncs the
+     * project config's `maxHasBeenInvoked` state. Also configures the project
+     * to use the core contract's maximum invocations value, and not the local
+     * manual limit set on the minter.
+     * WARNING - If relying on the local maximum invocations value, BE SURE TO
+     * COLLECT REVENUES BEFORE CALLING THIS FUNCTION. Otherwise, the local
+     * maximum invocations value will be set to the core contract's maximum
+     * invocations value, and the auction will be able to be invoked again,
+     * potentially reducing the artist's revenue below sellout price.
+     * @param _projectId Project ID to set the maximum invocations for.
+     * @dev this enables gas reduction after maxInvocations have been reached -
+     * core contracts shall still enforce a maxInvocation check during mint.
+     */
+    function setProjectMaxInvocations(
+        uint256 _projectId
+    ) external onlyArtist(_projectId) {
```

Comment:
Given that this method is kinda a footgun, I'm inclined to say that we actually just make it a no-op for this minter.. WDYT @ryley-o ?

Are there weird downstream effects of doing so (breaks out artist UI in some weird way)?

Reply:
agreed, and this adds complexity to this minter's interface that isn't really useful for artists. pushed an update that removes this function 🙏